src/c_wrapper.jl

```diff
@@ -44,6 +44,13 @@ function SCS_solve(m::Int, n::Int, A::SCSVecOrMatOrSparse, b::Array{Float64},
         slack::Vector{Float64}=Float64[];
         options...)
 
+    T = SCS.Indirect # the default method
```

Comment:
At the level of the C wrapper, `Direct` and `Indirect` should be parameters to `SCS_solve` as below, not stuffed into `options`.

Reply:
I moved this default and options "cleaning" to `optimize!` in `SCSSolverInterface.jl`